src/lib/select/select.html

```diff
@@ -19,31 +19,31 @@
   cdk-connected-overlay
   cdkConnectedOverlayLockPosition
   cdkConnectedOverlayHasBackdrop
+  [cdkConnectedOverlayOpen]="true"
```

Comment:
My concern with this one is how it would perform on a page with a lot of selects. I wonder whether we couldn't get away with not rendering it on init, but doing it when the user focuses on a select or hovers over it. That way we should have a split second to get it into the DOM before the user decides to open the panel.

Reply:
Not mentioned here, but the second half of this change is going to be support the native `<select>` in `mat-form-field` for situations where `mat-select` is too heavy. The user can then decide to make the trade-off- of having the Material Design style panel or the much more performant native dropdown.